src/configurations/destinations/zoho/ui-config.json

```diff
@@ -140,7 +164,15 @@
                     "type": "redirect",
                     "redirectGroupKey": "multiSelectFieldLevelDecision",
                     "label": "Module Specific Multi Select Pick List Field Action Mapping",
-                    "note": "Map Zoho Multi Select Pick List Field Action To Append or Replace"
+                    "note": "Map Zoho Multi Select Pick List Field Action To Append or Replace",
```

Comment:
Is the entire section of fields mapping getting hidden ? can you attach screenshots of destination page (with minimized sections) in the PR description

Reply:
No, it isn't all the section hidden. 
I have added some screenshots